File: src/python/ch01/1.2/l0/demo_basic_ops.py
def main():
    a, b = 478, 259
    print("Addition:", a, "+", b, "=", a + b)

    a, b = 503, 78
    print("Subtraction:", a, "-", b, "=", a - b)

    a, b = 214, 3
    print("Multiplication:", a, "*", b, "=", a * b)

    n, d = 47, 5
    q, r = divmod(n, d)
    print(f"Division: {n} / {d} -> quotient {q}, remainder {r}")


if __name__ == "__main__":
    main()

File: archived/v1/src/erlang/ch01/1.2/l0/src/demo_basic_ops.erl
-module(demo_basic_ops).
-export([main/0]).

main() ->
    A = 478, B = 259,
    io:format("Addition: ~p + ~p = ~p~n", [A, B, A + B]),

    A1 = 503, B1 = 78,
    io:format("Subtraction: ~p - ~p = ~p~n", [A1, B1, A1 - B1]),

    A2 = 214, B2 = 3,
    io:format("Multiplication: ~p * ~p = ~p~n", [A2, B2, A2 * B2]),

    N = 47, D = 5,
    Q = N div D, R = N rem D,
    io:format("Division: ~p / ~p -> quotient ~p, remainder ~p~n", [N, D, Q, R]).
